src/applications/validate-mhv-account/constants.js

```diff
@@ -17,4 +21,7 @@ export const MHV_ACCOUNT_LEVELS = {
   PREMIUM: 'Premium',
 };
 
-export const MHV_URL = 'https://www.myhealth.va.gov/mhv-portal-web/home';
+const envPrefix = eauthEnvironmentPrefixes[environment.BUILDTYPE];
+export const MHV_URL = hasSessionSSO()
+  ? `https://${envPrefix}eauth.va.gov/mhv-portal-web/eauth`
+  : 'https://www.myhealth.va.gov/mhv-portal-web/home';
```

Comment:
Since the `eauth.va.gov` part is used across files, it could be nice to have this value centralized somewhere as a base URL

Reply:
it's used in 2 places and I was waffling on where it best belonged so, adding this as a cleanup note to re-examine